docs/Deployment.md

```diff
@@ -46,9 +48,11 @@ juju config serial-vault api_keys=Heib2vah2aen3ai
 # Configure the admin service
 #   keystore_secret: part of the key used that is used to encrypt the stored data
 #   api_keys: the key that must be provided in the header of the web service requests
+#	csrf_auth_key: the key to protect server routing from csrf attacks
 #   (The keystore_secret and API key must be the same for the two services)
 juju config serial-vault-admin keystore_secret=uXeid2iy1Roo0Io0Beigae3iza5oechu
 juju config serial-vault-admin api_keys=Heib2vah2aen3aid
+juju config serial-vault-admin csrf_auth_key=2E6ZYnVYUfDLRLV/ne8M6v1jyB/376BL9ORnN3Kgb04uSFalr2ygReVsOt0PaGEIRuID10TePBje5xdjIOEjQQ==
```

Comment:
Not sure about whether we need quotes around the actually csrf_auth_key to make this work (because of '/' characters)